core/src/main/java/curso/exception/CursoFechaNoValidaException.java

```diff
@@ -0,0 +1,7 @@
+package curso.exception;
+
+public class CursoFechaNoValidaException extends RuntimeException {
+    public CursoFechaNoValidaException(String laFechaNoEsValida) {
```

Comment:
El nombre de la variable debe ser genérico. Probablemente el autocompletar de IntelliJ sugirió ese nombre a partir del String ejemplo. Pero revisar para que no quede así.
```suggestion
    public CursoFechaNoValidaException(String message) {
```
ℹ️  Aplica a las otras excepciones